packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3325,6 +3325,9 @@ describe('plugin-meetings', () => {
               options: {
                 meetingId: meeting.id,
               },
+              meetingJoinInfo: {
+                clientSignallingProtocol: CLIENT_SIGNALING_PROTOCOL.WEBRTC,
```

Comment:
Why do we not use this `CLIENT_SIGNALING_PROTOCOL.WEBRTC` for the main logic in `packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts`?